Assets/Scripts/Internal/PlayerGPS.cs

```diff
@@ -518,7 +518,7 @@ private void UpdateWorldInfo(int x, int y)
             currentClimateIndex = dfUnity.ContentReader.MapFileReader.GetClimateIndex(x, y);
             currentPoliticIndex = dfUnity.ContentReader.MapFileReader.GetPoliticIndex(x, y);
             climateSettings = MapsFile.GetWorldClimateSettings(currentClimateIndex);
-            if (currentPoliticIndex > 128)
+            if (currentPoliticIndex >= 128)
```

Comment:
This should be fine. In fact, I think you've fixed a much older bug into the bargain. The politic index is the value read from [POLITIC.PAK](https://en.uesp.net/wiki/Daggerfall:POLITIC.PAK) map. UESP states:

>For values from 128 through and including 233, subtract 128 from the value and to derive the Region Number for that pixel.

So >= 128 should be the correct solution here, with region numbers being zero-based in this case.

Thanks for the fix. :)